internal/pkg/sylog/sylog_test.go

```diff
@@ -0,0 +1,122 @@
+// Copyright (c) 2019, Sylabs Inc. All rights reserved.
+// This software is licensed under a 3-clause BSD license. Please consult the
+// LICENSE.md file distributed with the sources of this project regarding your
+// rights to use or distribute this software.
+
+// +build sylog
+
+package sylog
+
+import (
+	"strings"
+	"testing"
+)
+
+// TestSylogSetLevel to ensure the correct integer is returned
+func TestSylogSetLevel(t *testing.T) {
+	var levelTests = []struct {
+		name  string
+		level int
+	}{
+		{"SetLevelFatal", int(fatal)},
+		{"SetLevelError", int(error)},
+		{"SetLevelWarn", int(warn)},
+		{"SetLevelLog", int(log)},
+		{"SetLevelInfo", int(info)},
+		{"SetLevelVerbose", int(verbose)},
+		{"SetLevelVerbose2", int(verbose2)},
+		{"SetLevelVerbose3", int(verbose3)},
+		{"SetLevelDebug", int(debug)},
+	}
+
+	for _, tt := range levelTests {
+		t.Run(tt.name, func(t *testing.T) {
+			SetLevel(tt.level)
+
+			l := GetLevel()
```

Comment:
This line is unnecessary, we can just do `if GetLevel() == tt.level`

Reply:
The reason is because we want to save the variable and report the result back to the user - we use it twice.